docs/framework/guides/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md

```diff
@@ -595,6 +641,7 @@ export default class AbbreviationCommand extends Command {
 
 				writer.removeSelectionAttribute( 'abbreviation' );
 			} else {
+				// When the selection is non-collapsed, the code runs.
```

Comment:
Please make it a bit more descriptive,e.g. like here: https://github.com/ckeditor/ckeditor5-tutorials-examples/blob/9689e7af4903cb2a9c0f197e5a3d660dbbd4cf5f/abbreviation-plugin/part-3/abbreviation/abbreviationcommand.js#L101